sky/clouds/service_catalog/images/README.md

```diff
@@ -11,45 +11,61 @@ packer init plugins.pkr.hcl
 
 ## Generate Images
 ```bash
-export CLOUD=gcp
-export TYPE=gpu
+export CLOUD=gcp    # Update this
+export TYPE=gpu    # Update this
 export IMAGE=skypilot-${CLOUD}-${TYPE}-ubuntu
-
 packer build ${IMAGE}.pkr.hcl
 ```
 You will see the image ID after the build is complete.
 
+FYI time to packer build an image:
+
+| Cloud | Type | Approx. Time |
+|-------|------|------------------------|
+| AWS   | GPU  | 15 min          |
+| AWS   | CPU  | 10 min          |
+| GCP   | GPU  | 16 min          |
+| GCP   | CPU  | 5 min          |
+
 ### GCP
 ```bash
-# NEED UPDATE. Copy from the output of Packer build
-export IMAGE_NAME=skypilot-gcp-cpu-ubuntu-20241011003407 
+export IMAGE_NAME=skypilot-gcp-cpu-ubuntu-20241011003407  # Update this
 
 # Make image public
 export IMAGE_ID=projects/sky-dev-465/global/images/${IMAGE_NAME}
 gcloud compute images add-iam-policy-binding ${IMAGE_NAME} --member='allAuthenticatedUsers' --role='roles/compute.imageUser'
 ```
 
 ### AWS
+1. Generate images for all regions
 ```bash
-# Make the main image public
 export IMAGE_ID=ami-0b31b24524afa8e47   # Update this
-aws ec2 get-image-block-public-access-state --region us-east-1
-aws ec2 modify-image-attribute --image-id ${IMAGE_ID} --launch-permission "{\"Add\": [{\"Group\":\"all\"}]}"
 
-# Generate images for all regions and output a CSV file to be uploaded to SkyPilot Catalog
-TODO
+python aws_utils/image_gen.py --image-id ${IMAGE_ID} --processor ${TYPE}
 ```
+2. Add fallback images if any region failed \
+Look for "NEED_FALLBACK" in the output `images.csv` and edit. (You can use public [ubuntu images](https://cloud-images.ubuntu.com/locator/ec2/) as fallback.)
 
 ## Test Images
-Update the image ID in `sky/clouds/gcp.py` and run the test:
-```
+1. Minimal GPU test: `sky launch --image ${IMAGE_ID} --gpus=L4:1 --cloud ${CLOUD}` then run `nvidia-smi` in the launched instance.
+2. Update the image ID in `sky/clouds/gcp.py` and run the test:
+```bash
 pytest tests/test_smoke.py::test_minimal --gcp
 pytest tests/test_smoke.py::test_huggingface --gcp
 pytest tests/test_smoke.py::test_job_queue_with_docker --gcp
 pytest tests/test_smoke.py::test_cancel_gcp
 ```
 
-## Ship Images
+## Ship Images & Cleanup
 Create a PR to update [`SkyPilot Catalog`](https://github.com/skypilot-org/skypilot-catalog/tree/master/catalogs). 
 
-TODO: add PR examples.
\ No newline at end of file
+### GCP
+Example PR: [#86](https://github.com/skypilot-org/skypilot-catalog/pull/86)
+
+### AWS
+Example PR: [#89](https://github.com/skypilot-org/skypilot-catalog/pull/89)
+
+Then clean up the old images to avoid extra AMI storage fees.
+```bash
+python aws_utils/image_delete.py --tag ${TAG}
+```
```

Comment:
Should we do this before creating the new catalog? Otherwise, it will delete the newly created images?